repos.yaml

```diff
@@ -1344,3 +1344,10 @@ repositories:
     maintainers:
       - name: Rasool Safari
         email: safari@rasooll.com
+  - name: pulsar
```

Comment:
Why is this called `pulsar` instead of `kafkaesque`? Will Pulsar be the only thing in this repository?

Reply:
This is a Helm repo for Apache Pulsar, an open source data streaming and message queuing framework. We are the organization to support this Pulsar helm repo. Yes Pulsar will be the only thing in the given repo. Does this make sense?